src/main/java/org/cloudburstmc/api/Server.java

```diff
@@ -19,4 +30,28 @@
     PluginManager getPluginManager();
 
     GameRuleRegistry getGameRuleRegistry();
+
+    int getTick();
+
+    EventManager getEventManager();
+
+    boolean getAllowFlight();
+
+    PermissionManager getPermissionManager();
+
+    String getMotd();
```

Comment:
Some of these methods could be relegated to a configuration class.

Reply:
That sounds like a todo later PR situation